src/views/app/detail-panel/conversation-preview-panel.jsx

```diff
@@ -25,9 +25,14 @@ const MessagesComponent = ({ conversation }) => {
 	const messages = useSelector(selectMessages);
 	const conversationStatus = useSelector(selectCurrentFolderExpandedStatus)[conversationId];
 	const activeRoute = useCurrentRoute();
+	const searchInTrash = useMemo(
+		() => settings.prefs.zimbraPrefIncludeTrashInSearch === 'TRUE',
+		[settings]
+	);
 	const convMessages = useMemo(() => {
 		const msgs =
-			folderId !== FOLDERS.TRASH && activeRoute.id !== 'search'
+			(folderId !== FOLDERS.TRASH && activeRoute.id !== 'search') ||
```

Comment:
just a question ,I didn't check by running it but is the condition correct ? because the first and second are opposite of each other ,so the condition will always be true

Reply:
I don't understand your question, but yes, it runs